openpype/hosts/max/plugins/publish/validate_tyflow_data.py

```diff
@@ -0,0 +1,74 @@
+import pyblish.api
+from openpype.pipeline import PublishValidationError
+from pymxs import runtime as rt
+
+
+class ValidateTyFlowData(pyblish.api.InstancePlugin):
+    """Validate that TyFlow plugins or
+    relevant operators being set correctly."""
+
+    order = pyblish.api.ValidatorOrder
+    families = ["pointcloud", "tycache"]
+    hosts = ["max"]
+    label = "TyFlow Data"
+
+    def process(self, instance):
+        """
+        Notes:
+            1. Validate the container only include tyFlow objects
+            2. Validate if tyFlow operator Export Particle exists
+
+        """
+        report = []
+
+        invalid_object = self.get_tyflow_object(instance)
+        if invalid_object:
+            report.append(f"Non tyFlow object found: {invalid_object}")
+
+        invalid_operator = self.get_tyflow_operator(instance)
+        if invalid_operator:
+            report.append((
+                "tyFlow ExportParticle operator not "
+                f"found: {invalid_operator}"))
+        if report:
+            raise PublishValidationError(f"{report}")
+
+    def get_tyflow_object(self, instance):
+        invalid = []
+        container = instance.data["instance_node"]
+        self.log.info(f"Validating tyFlow container for {container}")
+
+        selection_list = instance.data["members"]
+        for sel in selection_list:
+            sel_tmp = str(sel)
+            if rt.ClassOf(sel) in [rt.tyFlow,
+                                   rt.Editable_Mesh]:
+                if "tyFlow" not in sel_tmp:
+                    invalid.append(sel)
+            else:
+                invalid.append(sel)
+
+        return invalid
+
+    def get_tyflow_operator(self, instance):
```

Comment:
missing docstrings

Reply:
Is `_summary_` supposed to be there?

Reply:
ah yes. I will add the docstring to this one.